test/evergreen/csuite_test_evg_task.template

```diff
@@ -1,4 +1,5 @@
   - name: {{task_name}}
+    tags: ["pull_request"]
```

Comment:
I don't know why we need these template files. They just duplicate the same code in the evergreen.yaml file. Are we using them correctly?

Reply:
When people add new make check directories or new csuite tests, it's expected some new code being added into evergreen.yml to cover those tests in Evergreen. The templates files are used to make people's life easier in crafting the evergreen config. Try run `s_evergreen` after adding a new test and you should see the output.